python/truenas_storage_monitor/config.py

```
@@ -103,27 +103,27 @@ def load_config(config_path: Optional[str] = None) -> Dict[str, Any]:
             if path.exists():
                 config_path = str(path)
                 break
         else:
             # No config file found, use defaults
             return get_default_config()
-    
+
     try:
         with open(config_path, "r") as f:
             config = yaml.safe_load(f)
     except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)